man/org.freedesktop.systemd1.xml

```diff
@@ -2503,6 +2503,8 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
       readonly t StartupCPUShares = ...;
       @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
+      readonly x CPUIdle = ...;
```

Comment:
this certainly should not be of type "x". It should either be a boolean, or a string. I understand it's actually a boolean in the kernel right now too, but exported as an integer for "future additions". But we should not expose a low-level concept like that. If it's an enum (i.e. "regular" + "idle") then expose it as string (because dbus doesn't have enums). But exposing this as integer is definitely wrong, because it doesn't really take arbitrary numeric values.